src/js/content_scripts/input-mapper.js

```diff
@@ -178,19 +183,23 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
      * tab/window.
      *
      * @param {String} visibilityStatus - The visibility status of the tab/window.
+     * @param {Object} configurationOptions - The configuration options for the
+     *                                        inputMapper component.
      *
      */
     gamepad.inputMapperManager = (function () {
         var inputMapperInstance = null;
 
-        return function (visibilityStatus) {
+        return function (visibilityStatus, configurationOptions) {
+            configurationOptions = configurationOptions || {};
+
             /**
              * Create an instance of the inputMapper when the tab/window is focused
              * again and start reading gamepad inputs (if any gamepad is connected).
              */
             if (visibilityStatus === "visible") {
-                // Obtain the saved configuration of the gamepad.
-                inputMapperInstance = gamepad.inputMapper();
+                // Pass the configuration options to the inputMapper component.
+                inputMapperInstance = gamepad.inputMapper(configurationOptions);
```

Comment:
Can you please leave a TODO here suggesting that we refactor to use a dynamic component here?  In general creating components dynamically like this is discouraged.  Using dynamic components would also make this specific issue much easier to address, as you would  have already included any configurable options as IoC references in the definition of the dynamic component.